pkg/asset/installconfig/ovirt/ovirt.go

```diff
@@ -0,0 +1,175 @@
+package ovirt
+
+import (
+	"fmt"
+	"sort"
+
+	ovirtsdk4 "github.com/ovirt/go-ovirt"
+	"gopkg.in/AlecAivazis/survey.v1"
+
+	"github.com/openshift/installer/pkg/types/ovirt"
+)
+
+// Platform collects ovirt-specific configuration.
+func Platform() (*ovirt.Platform, error) {
+	p := ovirt.Platform{}
+
+	err := survey.Ask([]*survey.Question{
+		{
+			Prompt: &survey.Input{
+				Message: "Enter oVirt's api endpoint URL",
+				Help:    "oVirt engine api url",
+				Default: "https://ovirt-engine-fqdn/ovirt-engine/api",
```

Comment:
This looks like an example. not a default. If it is not a default, just remove it. Default is not required.